tests/nested/manual/cloud-init-never-used-not-vuln/task.yaml

```diff
@@ -2,6 +2,12 @@ summary: |
     Test that cloud-init is no longer vulnerable on Ubuntu Core with the fix for
     CVE-2020-11933 in place.
 
+details: |
+    CVE-2020-11933 allowed anyone to present the device with cloud-info
+    meta-data, for example on a removable media, and reboot the machine to gain
```

Comment:
is it alright to make this exploitable implementation detail potentially public?

Reply:
this cve is already fixed in snapd, so it should be ok